commit ac83e58d1b9af68042433ae7ca8c9ff55b379770
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Fri Dec 27 15:43:14 2019 -0800

    Do not put generated *.so file to 'lib/'
    
    Keep the *.so files under extension directory only. 'lib/' is for ruby
    sources only.
    
    References:
      https://src.fedoraproject.org/rpms/rubygems/blob/master/f/operating_system.rb
      https://github.com/rubygems/rubygems/issues/749

diff --git a/lib/rubygems/defaults.rb b/lib/rubygems/defaults.rb
index d4ff4a26..4926f322 100644
--- a/lib/rubygems/defaults.rb
+++ b/lib/rubygems/defaults.rb
@@ -144,7 +144,7 @@ module Gem
   # Install extensions into lib as well as into the extension directory.
 
   def self.install_extension_in_lib # :nodoc:
-    true
+    false
   end
 
   ##
